pkgs/development/compilers/chicken/5/chicken.nix

```diff
@@ -26,8 +24,16 @@ stdenv.mkDerivation {
   # -fno-strict-overflow is not a supported argument in clang on darwin
   hardeningDisable = lib.optionals stdenv.isDarwin ["strictoverflow"];
 
+  enableParallelBuilding = true;
```

Comment:
```suggestion
```
Fails immediately and consistently on my system, there's a race condition in creation of chicken-config.h

```
chicken.h:48:11: fatal error: chicken-config.h: No such file or directory
```